README.md

```diff
@@ -15,27 +15,23 @@ import (
 )
 
 func main() {
-    f1.Execute()
+    f1.Scenarios().Execute()
 }
 ``` 
 
 This will give you a basic `f1` command line runner with the various running modes (see below) that are bundled with it. You can get more information about the various options available by simply running `go run main.go --help`
 
 ### Writing load tests
-Test scenarios consist of three stages: `Setup`, `Run` and `Teardown`. Setup is called once at the start of a test; this may be useful for generating resources needed for all tests, or subscribing to message queues. Run is called for every iteration of the test, often in parallel with multiple goroutines. Teardown is called once after all iterations complete. These Setup, Run and Teardown functions are defined as types in `f1`:
+Test scenarios consist of two stages: `Setup` and `Run`. Setup is called once at the start of a test; this may be useful for generating resources needed for all tests, or subscribing to message queues. Run is called for every iteration of the test, often in parallel with multiple goroutines. Teardown can be called once after all iterations complete to clean up the setup or/and after each iteration to clean up the run in LIFO order. These Setup and Run functions are defined as types in `f1`:
```

Comment:
This looks like `Setup` and `Run` have technical representations within `f1`, but they don't. Thus I would just refer to them as "Setup and run".

Reply:
Regarding Teardown, I would mention the fact that 

1. you may have multiple teardown callbacks for both setup and run
2. they are execute in a LIFO fashion

Reply:
It reads strange that "Setup and Run functions are defined as types in `f1`", and then seeing a `ScenarioFn` and a `RunFn`. Maybe worth pointing out that "Setup" actually means calling the `ScenarioFn` in the sense of "Please setup my scenario by calling the ScenarioFn"?